code/game/machinery/doors/windowdoor.dm

```diff
@@ -59,6 +59,7 @@
 	if(!open())
 		return
 	autoclose = TRUE
+	wires.ui_update()
```

Comment:
Errors whenever a windoor does anything.

It doesn't even have a wire datum, the flag up there lies.

Reply:
Good spot